pkgs/development/tools/rojo/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rojo";
+  version = "7.2.1";
+
+  src = fetchFromGitHub {
+    owner = "rojo-rbx";
+    repo = "rojo";
+    rev = "v${version}";
+    sha256 = "sha256-Kmq/lBwayYkFU4mbjExj7M9wpg59OkIiTc+2ZrwpuBc=";
+    fetchSubmodules = true;
+  };
+  cargoSha256 = "sha256-qx6Ja0DMe4cEmDSpovtY9T3+0nJS9XivR92K3UKgacE=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ];
+
+  meta = with lib; {
+    description = "Project management tool for Roblox";
+    longDescription = "Rojo is a tool designed to enable Roblox developers to use professional-grade software engineering tools.";
```

Comment:
Span the long descriptions to multiple lines, 80-char limit. 
It makes the long description more readable.

Reply:
> 80-char limit.

nixpkgs has no 80 characters line limit and it is stupid in 2022. As long as the line is not very long like ~120-180+ characters and it makes sense to split it up everything is fine.